BackendWebApp/src/main/java/uk/ac/ebi/biostd/webapp/application/rest/controllers/FileManagerResource.java

```diff
@@ -121,6 +145,23 @@ private String getRequiredPath(String requestUrl, String separator) {
         return path;
     }
 
+    private Map<String, String> getPaths(HttpServletRequest request, String resource) {
```

Comment:
I do not like to use a map header , IMHO we should create a class which represent this information. Also we can create a custom `ArgumentResolver` and register it to extract this logic of controller in specialized code.